keps/693-multikueue/README.md

```diff
@@ -231,11 +232,43 @@ has a valid configuration and at least one of its MultiKueueClusters is `Active`
 Will monitor the workloads in the management cluster and manage their MultiKueue specific
 AdmissionCheckStates.
 
-When distributing the workloads across clusters MultiKueue Workloads controller will first create
-the Kueue's internal Workload object. Only after the workload is admitted and other clusters
-are cleaned-up the real job will be created, to match the Workload. That gives the guarantee
-that the workload will not start in more than one cluster. The workload will
-get the annotation stating where it is actually running.
+When distributing workloads across clusters, the MultiKueue Workload Controller follows a two-phase
+process for a set of worker clusters selection.
+##### Cluster Nomination Phase
+The process of selecting worker clusters for a workload depends on the dispatching mode configured
+for the Dispatcher. MultiKueue supports three dispatching modes: AllClusters (`kueue.x-k8s.io/multikueue-dispatcher-all-at-once`),
+Incremental (`kueue.x-k8s.io/multikueue-dispatcher-incremental`), and External.
+
+* **AllClusters Mode**:  
+  In this mode, all available clusters are considered for dispatching at once, and the workload is copied to each worker cluster.
+
+* **Incremental Mode**:  
+  In this mode, clusters are nominated incrementally in rounds. In each round, a new cluster is added to the set of previously nominated clusters by updating the `.status.multiKueueNominatedCluster` field to include the new cluster. This allows the workload to be considered by an increasing set of clusters over time, rather than all at once.
```

Comment:
As we synced with @mszadkow, please move  `status.multiKueueNominatedCluster` into spec. 

Please also describe the API as in the following style https://github.com/kubernetes-sigs/kueue/tree/main/keps/2724-topology-aware-scheduling#admin-facing-api